CoastalResilienceNetwork/GeositeFramework
# Add method for plugins to clean-up post print, and demo functionality

## Overview

Provides a way for plugins to reset adjustments to the map for cases when a plugin adjusts the map during the print process. This functionality is demonstrated in the identify point plugin.

Connects #1008 

## Testing Instructions

- Move the map to somewhere that is not Philadelphia.
- Open the identify point plugin, and activate print.
- Click through the modal, and verify the map is centered on Philadelphia.
- Close the print preview dialog, and verify the map is back where you last centered it on.
